src/data/complex/exponential.lean

```diff
@@ -803,10 +809,17 @@ lemma cos_square : cos x ^ 2 = 1 / 2 + cos (2 * x) / 2 :=
 by simp [cos_two_mul, div_add_div_same, mul_div_cancel_left, two_ne_zero', -one_div]
 
 lemma cos_square' : cos x ^ 2 = 1 - sin x ^ 2 :=
-by { rw [←sin_sq_add_cos_sq x], simp }
+by rw [←sin_sq_add_cos_sq x, add_sub_cancel']
 
 lemma sin_square : sin x ^ 2 = 1 - cos x ^ 2 :=
-by { rw [←sin_sq_add_cos_sq x], simp }
+by rw [←sin_sq_add_cos_sq x, add_sub_cancel]
+
+lemma cos_sq_of_tan {x : ℂ} (hx : cos x ≠ 0) : cos x ^ 2 = 1 / (1 + tan x ^ 2) :=
```

Comment:
Not sure about the naming here and below. Usually `of` refers to a hypothesis. `cos_sq_eq_one_div_tan`? That's both long and imprecise...

Reply:
What about this?
```suggestion
lemma inv_one_add_tan_sq {x : ℂ} (hx : cos x ≠ 0) : (1 + tan x ^ 2)⁻¹ = cos x ^ 2 :=
```